src/main/kotlin/com/exactpro/th2/common/value/ValueUtils.kt

```diff
@@ -17,33 +17,70 @@
 package com.exactpro.th2.common.value
 
 import com.exactpro.th2.common.grpc.ListValue
+import com.exactpro.th2.common.grpc.ListValueOrBuilder
 import com.exactpro.th2.common.grpc.Message
+import com.exactpro.th2.common.grpc.MessageOrBuilder
 import com.exactpro.th2.common.grpc.NullValue.NULL_VALUE
 import com.exactpro.th2.common.grpc.Value
-import com.exactpro.th2.common.grpc.Value.KindCase.MESSAGE_VALUE
 import com.exactpro.th2.common.grpc.Value.KindCase.SIMPLE_VALUE
+import com.exactpro.th2.common.grpc.ValueOrBuilder
 import java.math.BigDecimal
 import java.math.BigInteger
 
 fun nullValue(): Value = Value.newBuilder().setNullValue(NULL_VALUE).build()
+fun listValue() : ListValue.Builder = ListValue.newBuilder()
 
 fun Value.getString(): String? = if (this.kindCase == SIMPLE_VALUE) this.simpleValue else null
 fun Value.getInt(): Int? = this.getString()?.toIntOrNull()
 fun Value.getLong(): Long? = this.getString()?.toLongOrNull()
 fun Value.getDouble(): Double? = this.getString()?.toDoubleOrNull()
 fun Value.getBigInteger(): BigInteger? = this.getString()?.toBigIntegerOrNull()
 fun Value.getBigDecimal(): BigDecimal? = this.getString()?.toBigDecimalOrNull()
-fun Value.getMessage(): Message? = if (this.kindCase == MESSAGE_VALUE) this.messageValue else null
+fun Value.getMessage(): Message? = if (hasMessageValue()) this.messageValue else null
+fun Value.getList() : List<Value>? = if (hasListValue()) this.listValue.valuesList else null
 
-fun String.toValue(): Value = Value.newBuilder().setSimpleValue(this).build()
+fun Value.Builder.updateList(updateFunc: ListValue.Builder.() -> ListValueOrBuilder?) : Value.Builder = apply { if (hasListValue()) updateOrAddList(updateFunc) else throw NullPointerException("Can not find list value") }
```

Comment:
Maybe change it like this:
```suggestion
fun Value.Builder.updateList(updateFunc: ListValue.Builder.() -> ListValueOrBuilder?) : Value.Builder = apply { check(hasListValue()) { "Can not find list value" }; updateOrAddList(updateFunc) }
```